release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2325,12 +2328,9 @@ This script removes unauthenticated subjects from the following cluster role bin
 No workaround exists for this issue.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=2100860[*BZ#2100860*])
 
-* On a newly installed {product-title} {product-version} cluster, platform monitoring alerts do not have the `openshift_io_alert_source="platform"` label.
-This issue does not affect clusters upgraded from a previous minor version.
-No workaround exists for this issue.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=2103127[*BZ#2103127*])
+* On a newly installed {product-title} {product-version} cluster, platform monitoring alerts do not have the `openshift_io_alert_source="platform"` label. This issue does not affect clusters upgraded from a previous minor version. There is currently no workaround exists for this issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2103127[*BZ#2103127*])
 
-* In the {rh-openstack-first}, a potential issue can affect Kuryr when the ports pools are populated with a variety of bulk port creation requests made at the same time. During these bulk requests, if the IP allocation for one of the IP addresses fails, the Neutron retries the operation for all ports. This issue was resolved in a previous errata; however, Red Hat suggests setting a small value for the ports pools batch to avoid large bulk port creation requests. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2024690[*BZ2024690*])
+* In the {rh-openstack-first}, a potential issue can affect Kuryr when the ports pools are populated with a variety of bulk port creation requests made at the same time. During these bulk requests, if the IP allocation for one of the IP addresses fails, the Neutron retries the operation for all ports. This issue was resolved in a previous errata; however, Red Hat suggests setting a small value for the ports pools batch to avoid large bulk port creation requests. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2024690[*BZ#2024690*])
```

Comment:
Recommend and its cousin suggest are frowned upon.
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#r
https://redhat-documentation.github.io/supplementary-style-guide/#_r